jobsdb/backup.go

```diff
@@ -68,7 +70,6 @@ func (jd *HandleT) backupDSLoop(ctx context.Context) {
 		opID := jd.JournalMarkStart(backupDSOperation, opPayload)
 		err = jd.backupDS(ctx, backupDSRange)
 		if err != nil {
-			stats.Default.NewTaggedStat("backup_ds_failed", stats.CountType, stats.Tags{"customVal": jd.tablePrefix, "provider": config.GetString("JOBS_BACKUP_STORAGE_PROVIDER", "S3")}).Increment()
```

Comment:
I had to remove this here because at the point where we are making this stat we can't determine the `provider` tag. This will be different for each workspace which was previously taken from the env.